python/src/block/DelayedNeutronPrecursorData.python.cpp

```diff
@@ -0,0 +1,166 @@
+// system includes
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+// local includes
+#include "ACEtk/block/DelayedNeutronPrecursorData.hpp"
+#include "views.hpp"
+#include "definitions.hpp"
+
+// namespace aliases
+namespace python = pybind11;
+
+namespace block {
+
+void wrapDelayedNeutronPrecursorData( python::module& module, python::module& ) {
+
+  // type aliases
+  using Block = njoy::ACEtk::block::DelayedNeutronPrecursorData;
+
+  // wrap views created by this block
+
+  // create the block
+  python::class_< Block > block(
+
+    module,
+    "DelayedNeutronPrecursorData",
+    "Delayed neutron precursor group data\n\n"
+    "The DelayedNeutronPrecursorData class contains the prrcursor group's decay\n"
+    "constant and the associated probability as a function of incident energy."
+  );
+
+  // wrap the block
+  block
+  .def(
+
+    python::init< double, std::vector< double >, std::vector< double > >(),
+    python::arg( "constant" ), python::arg( "energies" ),
+    python::arg( "probabilities" ),
+    "Initialise the block assuming linear interpolation\n\n"
+    "Arguments:\n"
+    "    self             the block\n"
+    "    constant         the precursor group's decay constant\n"
+    "    energies         the energy values\n"
+    "    probabilities    the probability values"
+  )
+  .def(
+
+    python::init< double,
+                  std::vector< long >, std::vector< long >,
+                  std::vector< double >, std::vector< double > >(),
+    python::arg( "constant" ),
+    python::arg( "boundaries" ), python::arg( "interpolants" ),
+    python::arg( "energies" ), python::arg( "xs" ),
+    "Initialise the block\n\n"
+    "Arguments:\n"
+    "    self             the block\n"
+    "    constant         the precursor group's decay constant\n"
+    "    boundaries       the interpolation range boundaries\n"
+    "    interpolants     the interpolation types for each range\n"
+    "    energies         the energy values\n"
+    "    probabilities    the probability values"
```

Comment:
`probabilities` passed to arg `xs`.

Reply:
Yes, it should be probabilities. Tests do not pick this up because this is another one of those ctors with interpolation data.